### DISCUSSION TITLE
Using an elevation layer without rendering it

### DISCUSSION THREAD

**Discussion by keince** [pelicanmapping/osgearth#2096] [Support Q&A] [answered]
This may be a dumb question but is it possible to load an elevation layer into a mapNode to use as a sample query mechanism and yet not render it on the screen?

We have a 2D map display in where we want to query for elevation at a given location but we don't want to actually render the elevation on the map display.

Thanks for any help or feedback...

**Comment by jasonbeverage** [comment, marked as answer]
I think you'd need to create a separate Map that contained your elevation layer only (different from the one you are rendering) and use the ElevationPool on that secondary map to do the height queries.

**Reply by keince** [reply]
Thank you Jason. That is what I was looking for.
